### DISCUSSION TITLE
Date and time on old forum

### DISCUSSION THREAD

**Discussion by krasnoshtan** [sumatrapdfreader/sumatrapdf#2711] [General]
Date and time of creating/editing is not present on [old forum](https://forum.sumatrapdfreader.org/). It could be important, especially if person comes from search results. And reactions counters is also important for keeping the vibe of conversation.
Also, I would like to download my data from it.

**Comment by kjk** [comment]
It was a lot of work to even get this archive up. No further work is planned.
